lib/crasher.js

```diff
@@ -33,19 +33,35 @@ function logCrashAndExit (logPath, errorInfo, checkout, stackTrace, err) {
   }
 }
 
-function setupCrashHandlers (processName, swap) {
+function printAndExit (errorInfo, stackTrace, err) {
+  // Electron does not play well with process.exit, so use app.exit instead
+  const runContext = isBare ? global.Bare : (global.process.versions.electron ? require('electron').app : global.process)
```

Comment:
some of this definitions are the same as logCrashAndExit, can we dedup?

Reply:
updated